.ado/pipelines/config/configuration.yaml

```diff
@@ -42,6 +42,17 @@ variables:
 - name: 'healthserviceImageName'
   value: 'alwayson/healthservice'
 
+# Embedded load testing (locust) configuration
```

Comment:
```suggestion
# Embedded load testing configuration
```

So we wont forget to remove it when we move to ALT ;)

Reply:
no sure if these parameters are still applicable then but ok for me